dto/src/main/java/ca/on/oicr/gsi/runscanner/dto/NotificationDto.java

```diff
@@ -271,7 +271,6 @@ public boolean equals(Object obj) {
         && Objects.equals(this.containerModel, other.containerModel)
         && Objects.equals(this.healthType, other.healthType)
         && Objects.equals(this.laneCount, other.laneCount)
-        && Objects.equals(this.metrics, other.metrics)
```

Comment:
If this is removed from `equals`, it should be removed from `hashCode` too, but why remove it from `equals`?